components/kyma-environment-broker/internal/process/deprovisioning/manager.go

```diff
@@ -70,9 +70,15 @@ func (m *Manager) Execute(operationID string) (time.Duration, error) {
 	}
 	operation := *op
 
-	var when time.Duration
-	logOperation := m.log.WithFields(logrus.Fields{"operation": operationID, "instanceID": operation.InstanceID})
+	pp, err := operation.GetProvisioningParameters()
+	if err != nil {
+		m.log.Errorf("while getting ProvisioningParameters from operation id %q: %s", operation.ID, err)
+		return 3 * time.Second, nil
```

Comment:
imo we shouldn't retry there because the error - if exist, will never be temporary.